lang/src/jmh/kotlin/org/partiql/jmh/utils/BenchmarkConstants.kt

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2022 Amazon.com, Inc. or its affiliates.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at:
+ *
+ *      http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ */
+
+package org.partiql.jmh.utils
+
+internal const val FORK_VALUE_RECOMMENDED: Int = 2
+internal const val MEASUREMENT_ITERATION_VALUE_RECOMMENDED: Int = 10
+internal const val MEASUREMENT_TIME_VALUE_RECOMMENDED: Int = 1
+internal const val WARMUP_ITERATION_VALUE_RECOMMENDED: Int = 5
+internal const val WARMUP_TIME_VALUE_RECOMMENDED: Int = 1
```

Comment:
I see a quite large fluctuation in the  example page, especially for commit 6d22a1
![image](https://user-images.githubusercontent.com/107505258/206008345-34a07235-25c9-4641-a3da-54e8d79d0c81.png).

I could not quite related the jump to the code changed, maybe play around with the parameter here to see if the result can be stabilized a little more? 

See here: https://stackoverflow.com/questions/43763530/how-to-choose-jmh-configuration-parameters